package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "mylibrary",
+  "version": "1.0.0",
+  "description": "App allow to user to create his own library, by adding books that have been read to their own list.",
+  "main": "index.js",
+  "dependencies": {
+    "bcryptjs": "^2.4.3",
+    "cookie": "^0.3.1",
+    "eslint": "^5.10.0",
```

Comment:
It would be great to quickly set eslint up first so your files are formatted correctly on all your computers 👍

Reply:
You can do this is another PR, just a note.